ingest/cell_metadata.py

```diff
@@ -147,15 +166,16 @@ def validate_type_annotations(self):
         annots = []
         for t in self.metadata_types[1:]:
```

Comment:
Can you add a comment of what's being done here at line 168?

Reply:
added comments in d86fc7c
        # skipping the TYPE keyword, iterate through the types
        # collecting invalid type annotations in list annots

also a little further down, for invalid value of blank space:
                # if the value is a blank space, store a higher visibility
                # string for error reporting